### ISSUE TITLE
Line wrapping in make-pot

### ISSUE THREAD

**Issue by Mr-Alexander** [wp-cli/i18n-command#216]
There is a different in line wrapping when using `makepot.php` vs `wp cli make-pot`.

WP CLI methods does not wrap at all.

**Comment by swissspidy** [comment]
That is correct, we don't currently do that, and there's no real reason for it. It makes no difference.

**Comment by Mr-Alexander** [comment]
Thanks for looking into this. It does make a difference :)

It would be nice to have that as an optional argument, for backward compatibility.

The reason is, for a file that has lots of existing translations, and if you want to change a small translation, the difference would include the line wrapping as well and then it would look like the entire file has been changed, where as only a small translation has been updated.

Hope it is not too much to ask for, either way, thanks for the reply.

**Comment by swissspidy** [comment]
Are you storing the POT files in version control or something? 

As for implementing this, it would require an upstream change in the gettext library we use: https://github.com/php-gettext/Gettext/issues/114

I looked into it two years ago and it wasn't really straightforward back then.

But if someone finds the time to address it, we could easily adopt it.

**Comment by kkmuffme** [comment]
Definitely against it. Line wrapping is deprecated. If this is implemented, then only as an optional flag

**Comment by swissspidy** [comment]
Closing for now since this is still blocked by https://github.com/php-gettext/Gettext/issues/114

It can be reopened for consideration once that one is fixed.
